openfpga_flow/benchmarks/micro_benchmark/adder_8/adder_8.eblif

```diff
@@ -0,0 +1,39 @@
+# Generated by Yosys 0.9+2406 (git sha1 e6ff764e, clang 12.0.0 -fPIC -Os)
+
+.model adder_8
+.inputs a[0] a[1] a[2] a[3] a[4] a[5] a[6] a[7] b[0] b[1] b[2] b[3] b[4] b[5] b[6] b[7] cin
+.outputs cout sum[0] sum[1] sum[2] sum[3] sum[4] sum[5] sum[6] sum[7]
+.names $false
+.names $true
+1
+.names $undef
+.subckt adder_lut4 in[3]=$false in[2]=sum_frac_lut4_lut4_out_lut2_out_carry_follower_a_cout[7] in[1]=$false in[0]=$false lut2_out[1]=cout_frac_lut4_lut4_out_lut2_out[1] lut2_out[0]=cout_frac_lut4_lut4_out_lut2_out[0] lut4_out=cout
+.param LUT 0110100101100001
+.subckt adder_lut4 in[3]=$false in[2]=sum_frac_lut4_lut4_out_lut2_out_carry_follower_a_cout[6] in[1]=a[7] in[0]=b[7] lut2_out[1]=sum_frac_lut4_lut4_out_lut2_out[1] lut2_out[0]=sum_frac_lut4_lut4_out_lut2_out[0] lut4_out=sum[7]
```

Comment:
Please re-generate this blif file using the latest yosys in OpenFPGA/yosys as that does not create this model - adder_lut4

Reply:
@lnsharma Yes. This is the next step to go. Currently, eblif is not supported in the openfpga flow. And that is why regression tests failed. Once it is clear, I can use the yosys + vpr flow.